Formula/csound.rb

```diff
@@ -0,0 +1,63 @@
+class Csound < Formula
+  desc "Sound and music computing system"
+  homepage "https://csound.com"
+  url "https://github.com/csound/csound/archive/6.12.2.tar.gz"
+  sha256 "39f4872b896eb1cbbf596fcacc0f2122fd3e5ebbb5cec14a81b4207d6b8630ff"
+
+  depends_on "cmake" => :build
+
```

Comment:
Please remove this empty line.

Reply:
I think this means the line below `depends_on "cmake" => :build`. This is now gone.